unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -1186,5 +1194,7 @@ public static void RequestUserProfile(string userId)
         }
 
         public static void ReportAvatarFatalError() { SendMessage("ReportAvatarFatalError"); }
+
+        public static void SendGenericAnalytic(string eventName, Dictionary<object, object> data) { SendMessage("GenericAnalytic", new GenericAnalyticPayload { eventName = eventName, data = data }); }
```

Comment:
There is already a `ReportAnalyticsEvent` method in this class.
Is replacing/removing that planned for a following PR?

Reply:
I didnt know about that method. Gonna dig a bit further, maybe we can reuse this and the `kernel` PR is not even needed.

Reply:
You were right @pbosio , the existent endpoint works as well here. I'm gonna use it!